hescorehpxml/schemas/hescore_json.schema.json

```
@@ -389,72 +411,65 @@
                                         ]
                                     }
                                 },
+                                "required": [
+                                    "roof_name",
+                                    "roof_type",
+                                    "roof_assembly_code",
+                                    "roof_color"
+                                ],
                                 "allOf": [
                                     {
                                         "if": {
                                             "properties": {
-                                                "roof_area": {"exclusiveMinimum": 0}
+                                                "roof_type": {"enum": ["vented_attic"]}
```

The use of `const` vs `enum` with a single value is inconsistent. Probably we should use just `const` unless there is a good reason.

---

So, would it look like this?

```suggestion
                                                "roof_type": {"const": "vented_attic"}
```